ui/cluster-admin/src/components/ClusterDetail.tsx

```diff
@@ -37,7 +37,7 @@ const ClusterDetail: React.FC<any> = () => {
   }, [clusterDetails.cluster?.metadata?.creationTimestamp])
 
   const onPanelClose = () => {
-    setShowClusterDetails(false)
+    setShowClusterDetails(undefined)
```

Comment:
We need to also `null` the `cluster` prop in `clusterDetails` here to make sure the `active` state works correctly

Reply:
So we need `setClusterDetails` from `useStore()` and set it here